src/index.ts

```diff
@@ -1,15 +1,25 @@
-import express, { Request, Response } from 'express';
+import express, { Request, Response} from 'express';
 import path from 'path';
+import { createTrip, getTrip }  from '../database/mongoose'
+
 
 const app = express();
 
+app.use(express.json())
+
 app.use(express.static(path.join(__dirname, '../../public')));
 
+app.get('/get-trip', getTrip)
```

Comment:
later it may be a good idea to split this out of index.ts with an express router